README.md

```diff
@@ -1,69 +1,39 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Climb Time 
 
-## Available Scripts
+## See it Live! 
+Climb Time on Github Pages
 
-In the project directory, you can run:
+## Set-Up Front-End:
+- Clone this repo
+- Run npm install from the root directory
+- Run npm start and visit localhost:3000 in your browser
 
-### `npm start`
+## Background and Information:
+There are a few websites that are dedicated to the climbing community with varying amounts of detailed information. Typically as you try to set up your weekend excursion you are talking with a friend going through a local guidebook, with outdated trailhead information and postit notes marking the pages that you want to remember. We thought there needs to be an easier option for people to find and track the climbs here in Colorado. 
```

Comment:
Rewording: We thought there needs to be an easier option for people to find and track the climbs here in Colorado. >>> We wanted there to be an easier way for people to find and track climbs here in Colorado so we developed this app that helps simplify that process.